.travis.yml

```diff
@@ -12,6 +12,7 @@ env:
 install:
     - sudo pip install tox
     - sudo pip install coveralls
+    - ./build_influxdb_server.sh || true
```

Comment:
Cause wasn t sure of need of command to return exit code of true (0) to continue test run..